app/controllers/resumes_controller.rb

```diff
@@ -40,18 +40,25 @@ def random
       return
     end
 
-    resumes = Resume.where(public: true).where.not(user_id: current_user.id).all
-    resumes = resumes.select {|resume| resume.user.points > 0}
-    resumes = resumes.select {|resume| resume.user.industry == current_user.industry}
-    resumes = resumes.select {|resume| resume.reviews.none? {|review| review.user_id == current_user.id }}
```

Comment:
we could keep the select to maintain the condition, if `.sample` loads everything anyways?